Create esphome_wled_controller.yaml

SevenZero0ne opened the pull request:
This blueprint will automatically turn on your esphome or wled controller while chamber light is on.
It will then change colors depending on that status of the printer.

i have not tested wled but it should work, im using esphome.

i feel like this could still use some work, but it has been working good enough for me, so i thought i would share for others to use and/or contribute to it.
sorry if the .yaml is kind of a mess, first blueprint i've made.
![%pn_13a79vVpax](https://github.com/user-attachments/assets/989fea35-23a4-4ea6-906f-7541b42cca23)

SevenZero0ne committed:
Create esphome_wled_controller.yaml

This blueprint will automatically turn on your esphome or wled controller while chamber light is on.

It will then change colors depending on that status of the printer.

i have not tested wled but it should work, im using esphome

SevenZero0ne committed:
Rename esphome_wled_controller.yaml to bambu_nr_esphome_wled_controller.yaml

Changed name of file to match other blueprints

SevenZero0ne committed:
Update bambu_nr_esphome_wled_controller.yaml

SevenZero0ne committed:
Update bambu_nr_esphome_wled_controller.yaml

changed the filter on the controllable light from device to entity

WolfwithSword commented:
Sorry this took a bit to get to - if you can, add a quick small section of it in [this readme](https://github.com/WolfwithSword/Bambu-HomeAssistant-Flows/blob/main/files/homeassistant/blueprints/nodered/README.md) with yourself marked as the author, and the easy click button to install the blueprint, and I'll merge it in!

SevenZero0ne committed:
Update bambu_nr_esphome_wled_controller.yaml

changed the effects from  a predefined list of names to text box.
you just type the name of the effect your device supports.
there is way to many effects that wled & esphome can have to list them all in a predefined list.

SevenZero0ne committed:
Update bambu_nr_esphome_wled_controller.yaml

some how had a typo & removed a bunch of words with effect at the end

SevenZero0ne committed:
Update bambu_nr_esphome_wled_controller.yaml

same as last

SevenZero0ne committed:
Update README.md

SevenZero0ne committed:
Update README.md

SevenZero0ne committed:
Update bambu_nr_esphome_wled_controller.yaml

SevenZero0ne committed:
Update README.md

SevenZero0ne committed:
Update README.md
